e2e/e2e_docker_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/GoogleContainerTools/kpt/pkg/test/runner"
 )
 
+// These e2e tests currently fail. When pipeline is implemented they should pass.
 func TestPipeline(t *testing.T) {
 	runPipelineTests(t, "../internal/pipeline/examples/")
```

Comment:
Can we rename `examples` to `testdata` ? to make it more idiomatic.

Reply:
done

Reply:
I see the files moved to `configdata` below, but this line still refers to `examples` dir.

Reply:
fixed, thanks for the catch